osu.Game/Overlays/Wiki/Markdown/WikiMarkdownParagraph.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using Markdig.Syntax;
+using Markdig.Syntax.Inlines;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers.Markdown;
+using osuTK;
+
+namespace osu.Game.Overlays.Wiki.Markdown
+{
+    public class WikiMarkdownParagraph : MarkdownParagraph
+    {
+        private readonly ParagraphBlock paragraphBlock;
+
+        public WikiMarkdownParagraph(ParagraphBlock paragraphBlock)
+            : base(paragraphBlock)
+        {
+            this.paragraphBlock = paragraphBlock;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            MarkdownTextFlowContainer textFlow;
+            InternalChild = textFlow = CreateTextFlow();
+            textFlow.AddInlineText(paragraphBlock.Inline);
```

Comment:
This code is already being run by the base class and doesn't need to exist here.
```suggestion
```

Reply:
Will do.

Reply:
Hmm after I look it again, those code can't just be removed.

I'll reimplement it with a bit different approach.